enterprise_access/apps/customer_billing/stripe_event_types.py

```diff
@@ -0,0 +1,257 @@
+"""
+https://github.com/stripe/stripe-python/blob/40a8d7e0/stripe/_event.py#L150
+"""
+from typing_extensions import Literal
+
+# List must exactly match the one from the stripe library source code.
+StripeEventType = Literal[
```

Comment:
Is there a reason to not import this from stripe?

Reply:
The stripe python bindings appear to be auto-generated, and do not internally type hint the "event_type" variable using a type hint "alias", instead opting to inject this long list wherever needed.  Non-DRYness probably doesn't matter for them because their code doesn't need to be DRY if a client generator tool is the only thing writing it.

As a result of the tool not generating a type hint alias, there's none available for me to use.  Yes, technically I could introspect using `stripe.Event.__annontations__['type']`, but all the static python type checkers only do static code evaluation, and do not actually launch a runtime which would be needed to perform dictionary lookups.

Reply:
But great question, and it should feel bad to do this.